### PR TITLE
add titles to pages

### PR DESCRIPTION
**Pull request by johndeyrup**
I am pretty inexperienced when it comes to seo, but not very much of our traffic is coming through search. Most of the traffic is coming from referrals and direct links. As far as I understand SEO we want to get our site linked on a bunch of sites; enable [robots.txt](https://github.com/codeforboston/vaccinatema/pull/21) and provide good titles that relate to COVID. If anyone has any other ideas happy to hear them.

Screen Shot 2021-02-10 at 7 43 45 PM
Screen Shot 2021-02-10 at 7 43 53 PM
Screen Shot 2021-02-10 at 7 43 57 PM
Screen Shot 2021-02-10 at 7 46 45 PM

### CODE REVIEW

**Review by mzagaja — APPROVED** [review]
This looks like a good improvement to me! Ready to merge.

File: views/FAQ.ejs [hunk 1]
```diff
@@ -5,11 +5,11 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="A site to help Massachusetts residents find a vaccine">
     <meta name="author" content="MA Volunteers">
+    <meta property="og:title" content="COVID-19 Vaccine FAQ" />
     <link rel="shortcut icon" type="image/x-icon" href="ma_logo.png" />
-    <meta property="og:title" content="Helping Massachusetts residents get vaccinated" />
     <meta property="og:image" content="ma_logo.png"/>
 
-    <title>Vaccinate MA</title>
+    <title>COVID-19 FAQs</title>
```
Comment (hblumberg) [inline views/FAQ.ejs]: I think the FAQs is more focused on questions about Vaccinate MA so this title might be a bit confusing? But totally fine if we want to prioritize getting COVID-19 in there 😅

File: views/eligibility.ejs [hunk 2]
```diff
@@ -6,10 +6,10 @@
     <meta name="description" content="A site to help Massachusetts residents find a vaccine">
     <meta name="author" content="MA Volunteers">
     <link rel="shortcut icon" type="image/x-icon" href="ma_logo.png" />
-    <meta property="og:title" content="Helping Massachusetts residents get vaccinated" />
+    <meta property="og:title" content="COVID-19 Vaccine Eligibility" />
     <meta property="og:image" content="ma_logo.png"/>
 
-    <title>Vaccinate MA</title>
+    <title>COVID-19 Eligibility</title>
```
Comment (hblumberg) [inline views/eligibility.ejs]: Again, I think "Vaccine Eligibility" makes a bit more sense, but understood if we are character limited and want COVID-19 in there (:

File: views/search.ejs [hunk 3]
```diff
@@ -6,10 +6,10 @@
     <meta name="description" content="A site to help Massachusetts residents find a vaccine">
     <meta name="author" content="MA Volunteers">
     <link rel="shortcut icon" type="image/x-icon" href="ma_logo.png" />
-    <meta property="og:title" content="Helping Massachusetts residents get vaccinated" />
+    <meta property="og:title" content="COVID-19 Vaccine Search" />
     <meta property="og:image" content="ma_logo.png"/>
 
-    <title>Vaccinate MA</title>
+    <title>COVID-19 Site Search</title>
```
Comment (hblumberg) [inline views/search.ejs]: (Ditto vaccine vs. COVID-19)

**Review by hblumberg — APPROVED** [review]
I don't know anything about SEO, but seems reasonable to me!

File: views/eligibility.ejs (hunk 2 above)
Comment (johndeyrup) [reply views/eligibility.ejs]: Could do COVID-19 Vaccine Eligibility. Does that seem to wordy?

Comment (hblumberg) [reply views/eligibility.ejs]: Works for me if it's not too long for the tab! (:

File: views/FAQ.ejs (hunk 1 above)
Comment (johndeyrup) [reply views/FAQ.ejs]: Maybe this page should be called get involved or something along those lines.

**Pull request merged**